package.json

```diff
@@ -452,6 +453,11 @@
         "command": "SonarLint.ConnectToSonarQube",
         "title": "Connect To SonarQube",
         "category": "SonarLint"
+      },
+      {
+        "command": "SonarLint.ModifyConnection",
```

Comment:
So will SonarQube and SonarCloud modification use the same command? Just asking as in my branch I have two separate commands `SonarLint.EditSonarQubeConnection` and `SonarLint.EditSonarCloudConnection`

Reply:
I'm not sure, maybe it would be cleaner to have separate commands (even if they load the same webview ultimately). I'll make sure to use `SonarLint.EditSonarQubeConnection` here to make the merge more straightforward.